src/refract/JsonSchema.cc

```diff
@@ -0,0 +1,290 @@
+//
+//  refract/JsonSchema.cc
+//  librefract
+//
+//  Created by Thomas Jandecka on 16/02/2018
+//  Copyright (c) 2018 Apiary Inc. All rights reserved.
+//
+
+#include "JsonSchema.h"
+
+#include <cassert>
+#include <bitset>
+#include "Element.h"
+#include "../ElementData.h"
+#include "Utils.h"
+#include "../utils/log/Trivial.h"
+
+#include "VisitorUtils.h" // GetValue
+
+using namespace refract;
+using namespace schema;
+using namespace drafter::utils;
+using namespace drafter::utils::log;
+
+namespace
+{ // API Elements tools
+    so::String instanciate(const StringElement& e)
```

Comment:
instanciate - misspell?

Reply:
How is that a misspell?

Reply:
Not sure, I just asking. I mean it should be "instantiate"

Reply:
https://en.wiktionary.org/wiki/instanciate
You're right.